src/Storages/Kafka/StorageKafka.cpp

```diff
@@ -174,6 +178,92 @@ struct StorageKafkaInterceptors
     }
 };
 
+class ReadFromStorageKafkaStep final : public ISourceStep
+{
+public:
+    ReadFromStorageKafkaStep(
+        const Names & column_names_,
+        StoragePtr storage_,
+        const StorageSnapshotPtr & storage_snapshot_,
+        SelectQueryInfo & query_info,
+        ContextPtr context_)
+        : ISourceStep{DataStream{.header = storage_snapshot_->getSampleBlockForColumns(column_names_)}}
+        , column_names{column_names_}
+        , storage{storage_}
+        , storage_snapshot{storage_snapshot_}
+        , storage_limits{query_info.storage_limits}
+        , context{context_}
+    {
+    }
+
+    String getName() const override { return "ReadFromStorageKafka"; }
```

Comment:
Minor, but usually I use `ReadStorageKafka` for both class name and getName()

Reply:
I will fix this in my next PR where I make the same fix for `StorageFileLog`. Is that okay?